src/miniscript/satisfy.rs

```diff
@@ -109,36 +106,35 @@ pub trait Satisfier<Pk: MiniscriptKey + ToPublicKey> {
     }
 
     /// Assert whether an relative locktime is satisfied
-    fn check_older(&self, _: u32) -> bool {
+    fn check_older(&self, _: Sequence) -> bool {
```

Comment:
If you introduce a relative locktime type then it should probably be passed in here instead of `Sequence`.

Reply:
Thanks man, I've done that now in the demo PR.